.circleci/config.yml

```diff
@@ -441,18 +440,32 @@ jobs:
           paths:
             - ./docs/_build/_html
       - checkout
+      - run:
+          name: Set up version
+          command: |
+            THISVERSION=$( python get_version.py )
+            sed -i -E "s/'version'/'${CIRCLE_TAG:-$THISVERSION}'/" docs/conf.py
```

Comment:
seems like sphinx-versioning also does this replacement, so I'll probably have to roll back these lines